package.json

```diff
@@ -72,7 +72,7 @@
     "rimraf": "^2.5.4",
     "tslint": "latest",
     "tslint-test-config-non-relative": "file:test/external/tslint-test-config-non-relative",
-    "typescript": "^2.3.0"
+    "typescript": "next"
```

Comment:
would that mean that tslint API consumers would need to upgrade to 2.4.0-dev as well? I'm very wary of making this change; even if the compiled tslint typings continue to work with 2.3 in _this PR_, bumping to a nightly version opens up the door to future API breaks